circle.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       - run: yarn run test
       - run: yarn run pack
       - run: mkdir artifacts
-      - run: mv release/*.yml artifacts || true
-      - run: mv release/falcon-sql-client* artifacts
+      - run: for f in release/falcon-sql-client-*.tar.gz ; do mv $f artifacts/linux-falcon-v${f#release/falcon-sql-client-}; done
```

Comment:
`{f#release/falcon-sql-client-}`?

Reply:
it's a bash parameter substitution; e.g. see see https://www.tldp.org/LDP/abs/html/parameter-substitution.html

Reply:
shall I perhaps add a comment explaining this line moves the installer from the `release` folder to `artifacts`?

Reply:
Yes, that would be great actually.. !!